### PR TITLE
index cleanup - sponsor moved to gatsby-config

### PR DESCRIPTION
**Pull request by sahupr**
- index.js is cleaner now with the sponsor images moved to gatsby-config.js
- removal of the contact us section below the sponsor section

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -68,8 +68,134 @@ module.exports = {
         answer: `We have developed 4 tracks (impact areas) to assist you with building your idea. However, you're not 
         required to submit your hack to a track to win a prize!`
       }
-    ]
+    ],
+    sponsors_L: [
+      {
+        link: "https://cloud.google.com/",
+        target: "_blank",
+        rel: "noopener noreferrer",
```
Comment (devagrawal09) [inline gatsby-config.js:76]:
```suggestion
```
Pretty much the same in every sponsor object. Just get rid of them here and hardcode it in the template.

File: src/components/Sponsors.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+const SponsorCard = ({ className, children, style = {} }) => (
+  <div
+    className={`${className}`}
+  >
+    {children}
+  </div>
+);
+
+export default SponsorCard;
```
Comment (devagrawal09) [inline src/components/Sponsors.js:11]: This component quite literally doesn't do anything. Just remove it?

File: src/pages/index.js [hunk 3]
```diff
@@ -198,112 +183,35 @@ export default () => {
...
-                        className="px-10 mt-0 partner"
-                        alt="Centauri"
-                      />
-                    </a>
+                  {sponsors_L.map((spl) => (
+                    <SponsorCard key={spl.link} className="m-4 text-secondary bg-white">
+                      <a href={spl.link} target={spl.target} rel={spl.rel}>
+                      <img src={spl.src} style={{ height: spl.imageHeight }} className={spl.className} alt={spl.alt} />
+                      </a>
+                    </SponsorCard>
```
Comment (devagrawal09) [inline src/pages/index.js:191]:
```suggestion
                    <div key={spl.link} className="m-4 text-secondary bg-white">
                      <a href={spl.link} target={spl.target} rel={spl.rel}>
                      <img src={spl.src} style={{ height: spl.imageHeight }} className={spl.className} alt={spl.alt} />
                      </a>
                    </div>
```

Comment (devagrawal09) [reply src/pages/index.js:191]: SponsorCard doesn't really do anything, unless you have plans for it.

**Review by devagrawal09 — CHANGES_REQUESTED** [review]
Also it looks like all the sponsors arrays (L, M, S) are defined the same way and get similar treatment. So why not put them inside a bigger array, and instead of writing the same jsx logic 3 times, put that in another loop?

**Comment by sahupr** [comment]
There are <hr> between all of these different categories. Putting them in one loop won't let me do that.

File: gatsby-config.js (hunk 1 above)
Comment (sahupr) [reply gatsby-config.js:76]: Yup great suggestions I'll make the change.

File: src/components/Sponsors.js (hunk 2 above)
Comment (sahupr) [reply src/components/Sponsors.js:11]: I'll look into it.

Comment (sahupr) [reply src/components/Sponsors.js:11]: I think there needs to be a ja object, even an empty one for the graphql variable to work.

File: gatsby-config.js (hunk 1 above)
Comment (sahupr) [reply gatsby-config.js]: updated the PR with this update

File: src/pages/index.js (hunk 3 above)
Comment (sahupr) [reply src/pages/index.js]: check my comment above for the explanation

**Comment by sahupr** [comment]
White space has been optimized in the latest PR, looks fine on mobile too. Take a look.

File: src/pages/index.js (hunk 3 above)
Comment (devagrawal09) [reply src/pages/index.js]: what do you mean? I changed the `SponsorCard` to`div` and nothing changed. It seems to be working fine

**Review by omaralsayed — APPROVED** [review]

**Pull request merged**